submissions/Roophee/Popup_checkbox/index.html

```diff
@@ -0,0 +1,484 @@
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="shortcut icon" href="img/icons/favicon.png" type="image/x-icon">
+    <title>PopUp Kottans</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="header">
+            <nav class="nav">
+                <ul class="nav-list">
+                    <li class="nav__item"><a href="#" class="nav-list__link">
+                            <span>Mail</span>
+                        </a>
+                    </li>
+                    <li class="nav__item">
+                        <a href="#" class="nav-list__link">
+                            <span>Images</span>
+                        </a>
+                    </li>
+                    <li class="nav__item nav__item-popup-button ">
+                        <div class=" nav__button--focus2 ">
+                            <input type="checkbox" name="popup__checkbox" class="popup__checkbox-check"
+                                id="popup__checkbox">
+                            <label for="popup__checkbox" class="popup__checkbox-label ">
+                                <svg display="none" xmlns="http://www.w3.org/2000/svg">
+                                    <symbol id="some" viewBox="0 0 512  512 ">
+                                        <path d="M104 160a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M256 160a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M408 160a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M104 312a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M256 312a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M408 312a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M104 464a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M256 464a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                        <path d="M408 464a56 56 0 1 1 56-56a56.06 56.06 0 0 1-56 56z" />
+                                    </symbol>
+                                </svg>
+                                <svg class="popup__button-svg popup__button" xmlns="http://www.w3.org/2000/svg">
+                                    <use xlink:href="#some">
+                                    </use>
+                                </svg>
+                            </label>
+
+                            <div class="popup__hide-pannel1">
+                                <ul class="popup__items-list1">
+                                    <li class="popup__item">
+                                        <a href="#">
+                                            <svg display="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why you use two `svg` elements in every list item?

Reply:
Sorry, is it wrong? I saw one educational video on youtube about sprites. Lector did like that. And I copy his approach. 
This my bad. I`ll learn this more thoroughly.
Thanks.

Reply:
Using sprites means you have **one** file with couple of images/icons and you used it in different places, In your case, you have "a sprite" with one icon in every list item.

Reply:
I have a promlem. When I move all "sprites" in one file. The SVG icons were gone from my popup and when I call them by id in svg>use nothing happens.

Reply:
[Read more about it](http://dreamhelg.ru/2017/02/symbol-svg-sprite-detail-guide/)
If you don't want to fix it let me know I'll merge your work.

Reply:
Alexandr, merge my pull*. Thanks.